### PR TITLE
video tutorials added #258

### PR DESCRIPTION
**Pull request by ashish0910**
Created video tutorials for sugarizer as in #258 

playlist link - https://www.youtube.com/playlist?list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV

Individual videos link -
https://www.youtube.com/watch?v=y_JhWU8rRRo&index=2&list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV&t=9s

https://www.youtube.com/watch?v=c6SJHrv_RCY&index=3&list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV&t=0s

https://www.youtube.com/watch?v=NJ0r5BVNqoU&index=4&list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV&t=0s

https://www.youtube.com/watch?v=BlnY8NYlTRk&index=5&list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV&t=0s

https://www.youtube.com/watch?v=Pzm7gGGD1qQ&index=6&list=PLrlbN_j855LldrkC39FX7u_JOxlLb4pLV&t=0s

### CODE REVIEW

**Comment by llaske** [comment]
Nice work.
Few remarks:
- "npm install" is not need to run the browser version of Sugarizer, it's need only for the electron version. Not sure it's clear in your video. The current tutorial don't mention this possibility to avoid confuse users.
- Why do not create a tutorial video for each tutorial step? For example tutorial step 2 are in video 2 and 3. Part 4 is only a part of tutorial step 3.
- In part 4, not a good idea to run directly the activities/Pawn.activity/index.html file.

**Comment by ashish0910** [comment]
- I will fix that npm install thing .
- Actually I wanted to keep the length of tutorial small around 3 minutes that's why I splitted it .however we can do it like 2a) 2b) and rename the others in sync or we can have a new video ?
- Yes that part 4 thing is not good idea , I will fix that .

**Comment by ashish0910** [comment]
@llaske new videos has been created as per the remarks you mentioned .
Link to the playlist - https://www.youtube.com/playlist?list=PLrlbN_j855LkYhgtmeX-RaTTm-X5wBkc6

**Comment by ashish0910** [comment]
Please review @llaske

**Comment by ashish0910** [comment]
@llaske  Please review the tutorials , I have made the changes as per the remarks 
Link to the playlist - https://www.youtube.com/playlist?list=PLrlbN_j855LkYhgtmeX-RaTTm-X5wBkc6

**Comment by ashish0910** [comment]
@llaske please review .

**Comment by ashish0910** [comment]
@llaske can you please review this pr .

**Comment by llaske** [comment]
The part 3 is not the whole tutorial step 3, a part is missing and is done in the part 4. As I said, it's better to do one video for each step.
Why do you have errors in console in the part 4?
It will be better too if in part 4 you run the "New instance", "Existing instance" tes before writing in the datastore. Doesn't matter if the video is longer.
In part 5, there is a missing string, it's better to avoid doing mistake during the tutorial, it could bring confusion.
It will be better if you indent the code during tutorials, it will help to understand the code structure. At end of part 5, the code is not aligned at all.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ To run **Sugarizer Application on iOS**, download it on [Apple Store](https://it
 
 ![](images/applestore.png)
 
+**Watch Video Tutorials for Sugarizer Application on GNU Linux/Mac OS/Windows** go [here](https://www.youtube.com/playlist?list=PLrlbN_j855LkYhgtmeX-RaTTm-X5wBkc6)
```
Comment (llaske) [inline README.md]: Not the right place to add this. Will be better on the tutorial part.

**Pull request closed without merging**